test/parallel/test-http-parser.js

```diff
@@ -129,7 +129,7 @@ function expectBody(expected) {
 //
 {
   const request = Buffer.from(
-      'HTTP/1.1 200 OK' + CRLF +
+    'HTTP/1.1 200 OK' + CRLF +
```

Comment:
The rule is actually that the continuation indentation is ignored. So this is just as good:

```js
  const request = Buffer.from(
    'HTTP/1.1 200 OK' + CRLF +
    'Content-Type: text/plain' + CRLF +
    'Content-Length: 4' + CRLF +
    CRLF +
    'pong'
  );
```